site/content/documentation/reference/configuration.md

```diff
@@ -11,53 +11,51 @@ RDF4J repositories and SAIL configuration can be set up and changed by means of
 
 ## Repository configuration
 
-A Repository configuration consists of a single RDF subject of type `rep:Repository`. Typically this subject is a blank node (`[]` in Turtle syntax), and is assigned its configuration parameters through use of RDF properties.
+A Repository configuration consists of a single RDF subject of type `config:Repository`. Typically this subject is a blank node (`[]` in Turtle syntax), and is assigned its configuration parameters through use of RDF properties.
```

Comment:
maybe it is worth here in this page to have a paragraph stating that repository configuration schema namespaces have been unified and that legacy usages should be updated. Ideally we would also already have a migration guide summarizing the changes on a higher level? There could be a quick pointer to that description here

Reply:
Good point. Added an info banner and a short migration section. 

Fwiw I'm also considering that (for the Server and Console applications at least) we will want to add automatic migration tooling at some point - but that's out of scope for this PR.